conf/messages.cy

```diff
@@ -308,3 +308,15 @@ addedItems.heading.singular = You have given information for {0} item
 addedItems.heading.plural = You have given information for {0} items
 addedItems.addAnother = Do you want to give information for another item?
 addedItems.error.required = Select yes if you want to give information about another item
+
+
+addGiveReasonItemDamaged.title = Give more information about being sent damaged items
+addGiveReasonItemDamaged.heading = Give more information about being sent damaged items
+addGiveReasonItemDamaged.hint = Give more information (optional)
+addGiveReasonItemDamaged.checkYourAnswersLabel = addGiveReasonItemDamaged
```

Comment:
This is wrong but I can change it on the CYA PR as I'm refactoring some other messages stuff with that PR